frontend/src/pages/Main.jsx

```diff
@@ -1,10 +1,48 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import { Row } from "antd";
 import Header from "../Components/Header/Header";
 import Main from "../Components/Main/Main";
 import ApproveData from "../Components/ApproveData/ApproveData";
+import api from "../lib/api";
+import useNotification from "../util/useNotification";
+import { useCookies } from "react-cookie";
+import store from "../lib/store";
 
 const MainPage = () => {
+  const { notify } = useNotification();
+  const [cookies] = useCookies(["AUTH_TOKEN"]);
+
+  const [dashboardData, setDashboardData] = useState(null);
+  const [pendingData, setPendingData] = useState(null);
+  const [approvedData, setApprovedData] = useState(null);
+
+  useEffect(() => {
+    api
+      .get(`/form`, {
+        headers: {
```

Comment:
you don't need a header when you already have a token added to the `api` using `api.setToken(thetoken)`